tests/test_conn.py

```diff
@@ -204,3 +204,12 @@ def invoke_osserror(*a, **kw):
         with self.assertRaises(ConnectionError):
             yield from conn.send(request)
         self.assertEqual(conn.connected(), False)
+
+    @run_until_complete
+    def test_close_disconnects_connection(self):
+        asyncio.set_event_loop(self.loop)
```

Comment:
Don't do this in tests. It's not without a reason it's there, you probably didn't pass loop explicitly somewhere.